### ISSUE TITLE
Implement support for Forgejo comments and reactions on it

### ISSUE THREAD

**Issue by lbarcziova** [packit/ogr#883]
- [ ] implement methods for the classes inheriting from [`Comment`](https://github.com/packit/ogr/blob/31f2ff50bfa4583ffe108640666f8470c692bef0/ogr/abstract.py#L171) and [`Reaction`](https://github.com/packit/ogr/blob/31f2ff50bfa4583ffe108640666f8470c692bef0/ogr/abstract.py#L159)
- [ ] use the functionality from https://codeberg.org/harabat/pyforgejo/src/branch/main/pyforgejo/api/issue

Client usage docs: https://codeberg.org/harabat/pyforgejo#usage

---
*Part of #866*

**Comment by julietowah** [comment]
Hello, @lbarcziova 
I would love to work on this issue

**Comment by julietowah** [comment]
Ok thank you Am on it

**Comment by julietowah** [comment]
Hello, @lbarcziova i am done with implementing the methods for this issue but the building of the test image following the documentation is where i gut stuck.

i installed the ansible-bender but when i run `make build-test-image` am getting this error 

`make build-test-image
ansible-bender build --build-volumes /home/juliet/ogr:/src:Z -- ./recipe.yaml fedora:latest ogr
16:32:50.660 utils.py          ERROR  Resolved "fedora" as an alias (/etc/containers/registries.conf.d/shortnames.conf)
16:32:50.661 utils.py          ERROR  Trying to pull registry.fedoraproject.org/fedora:latest...
16:33:04.344 utils.py          ERROR  Getting image source signatures
16:33:04.348 utils.py          ERROR  Copying blob sha256:bfa20db43df552b1a9d230d4c98ed9c96893c8334b7af253dbcae4174b16718a
16:33:12.850 utils.py          ERROR  Copying blob sha256:bfa20db43df552b1a9d230d4c98ed9c96893c8334b7af253dbcae4174b16718a
16:33:40.052 utils.py          ERROR  Copying config sha256:3acda6fe1bd0793ef63e491cefc392410d34d76f91018b741b2d35415d98514e
16:33:40.057 utils.py          ERROR  Writing manifest to image destination
16:33:40.057 utils.py          ERROR  Storing signatures
16:33:44.260 buildah_builder.py ERROR  couldn't locate python interpreter, tried these paths: ('/usr/bin/python3', '/usr/local/bin/python3', '/usr/bin/python3.7', '/usr/bin/python37', '/usr/bin/python3.6', '/usr/bin/python36', '/usr/bin/python2', '/usr/local/bin/python2', '/usr/bin/python', '/usr/local/bin/python', '/usr/libexec/platform-python')
There was an error during execution: no python interpreter was found in the base image "fedora:latest", you can specify the path via CLI option --python-interpreter
make: *** [Makefile:9: build-test-image] Error 2`

please can you help me to setup the test image file

**Comment by LecrisUT** [comment]
`ERROR` in this case I believe reffers to `srderr` output screen which does not necessarily mean error. The last part describes your error, it couldn't find your system `python`. If you have a python venv setup make sure you activate it before running the make commands, otherwise can you check where your python interpreter is with
```console
$ which python3
```

PS: please format code blocks like
````markdown
```
Long list of code
With multiple lines
```
````

**Comment by nforro** [comment]
> If others are able to run it normally though there is an error somewhere before that in `ansible-blender`

No, it looks like starting with Fedora 40 the container images don't have `/usr/bin/python3` by default.

**Comment by nforro** [comment]
https://github.com/ansible-community/ansible-bender/issues/49

**Comment by julietowah** [comment]
@LecrisUT and @nforro thank you so much let me do that

**Comment by julietowah** [comment]
please @LecrisUT can you make out time to look  at my codebase i cant get the test right i think my setup is messed up 

following the documentation in https://github.com/packit/contributing i installed the dependencies maybe i missed a step because i encountered  this 
```
 ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov=ogr --cov-report=term-missing
  inifile: None
  rootdir: /src
```
when i run: make check-in-container

**Comment by LecrisUT** [comment]
It seems the image is missing `pytest-cov`. Can you add it [here](https://github.com/packit/ogr/blob/73798e394f6b8fc8471c1b0231cbb5b5930358cc/recipe.yaml#L29) and re-run `build-test-image`

**Comment by julietowah** [comment]
Yes i thought as much
thank you so much 
it worked

**Comment by julietowah** [comment]
Hello @lbarcziova i tried ti add mergeit label on my pr https://github.com/packit/ogr/pull/906#partial-pull-merging but it seems i don't have the necessary permission

**Comment by LecrisUT** [comment]
> Hello [@lbarcziova](https://github.com/lbarcziova) i tried ti add mergeit label on my pr [#906 (comment)](https://github.com/packit/ogr/pull/906#partial-pull-merging) but it seems i don't have the necessary permission

It first needs to be reviewed to check that everything is clean and dandy 🙂 Only the core team has permission to merge PRs, so be patient. I usually distract myself with another task while waiting for reviews.

A quick note, could you simplify the PR title. A general rule of thumb is that the PR title will show up as the commit message title so be tidy about it.
